.github/workflows/go.yml

```diff
@@ -51,12 +49,7 @@ jobs:
     - name: Run Test
       run: |
         go test -short ./...
-        foreach ($line in (Get-Content wix/pluginlist.txt) -as [string[]]) {
-            powershell -Command "go list $line"
-            if ($LASTEXITCODE -ne 0) {
-              exit 1
-            }
-        }
+      shell: bash
```

Comment:
Currently `shell` clause is not needed.

However the default shell on Windows runner is powershell so it will not work as expected results to run `go test -coverprofile=cover.out` (especially equal sign). I think it has a certain meaning to avoid a mistake.